Formula/cmocka.rb

```diff
@@ -15,12 +15,13 @@ class Cmocka < Formula
 
   def install
     args = std_cmake_args
+    args << "-DWITH_STATIC_LIB=ON" << "-DWITH_CMOCKERY_SUPPORT=ON" << "-DUNIT_TESTING=ON"
```

Comment:
i think it's important for binary releases, though, since it's always very difficult to perfectly replicate compile-time environments. running `make check`, `make test`, etc. assures that the tests the packages maintainers wrote still work in your environment, which they should.

if not, is it really a good idea to release a binary that seems to not run properly? a lot of times, i don't think anyone other than original maintainers can confirm whether failing tests are an issue or not, but releasing a binary releases prevent an end user from knowing that a binary is potentially misbehaving